bacs/src/main/java/com/adyen/checkout/bacs/internal/ui/DefaultBacsDirectDebitDelegate.kt

```diff
@@ -199,7 +201,7 @@ internal class DefaultBacsDirectDebitDelegate(
             shopperEmail = outputData.shopperEmailState.value,
             paymentMethod = bacsDirectDebitPaymentMethod,
             order = order,
-            amount = componentParams.amount.takeUnless { it.isEmpty },
+            amount = componentParams.amount.takeUnless { it == null },
```

Comment:
Could this be just `componentParams.amount`?

Reply:
Indeed, missed these few.